api/Document.json

```diff
@@ -8228,30 +8228,72 @@
           }
         }
       },
-      "policy": {
+      "featurePolicy": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/policy",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/featurePolicy",
           "support": {
-            "chrome": {
-              "version_added": "69",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#enable-experimental-productivity-features",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            },
-            "chrome_android": {
-              "version_added": "69",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#enable-experimental-productivity-features",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            },
+            "chrome": [
+              {
+                "version_added": "74",
+                "notes": "<a href='https://www.chromestatus.com/feature/5190687460950016'>Chrome Platform Status</a> incorrectly describes it as <code>document.policy</code>. It was renamed in <a href='https://crbug.com/917057'>bug 917057</a> before being shipped in <a href='https://crbug.com/703703#c22'>bug 703703</a>."
+              },
+              {
+                "version_added": "73",
+                "version_removed": "74",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-productivity-features",
+                    "value_to_set": "Enabled"
+                  }
+                ],
+                "notes": "It was renamed from <code>document.policy</code> to <code>document.featurePolicy</code> in <a href='https://crbug.com/917057'>bug 917057</a>."
```

Comment:
I think we can leave this note out, as it's already specified by the data.  (I'll only comment on this one, but this applies to all of the notes of the same message.)